packages/system-tests/src/spectron/application.ts

```diff
@@ -109,6 +109,10 @@ export class SpectronApplication {
     );
   }
 
+  public waitUI(): Promise<any> {
```

Comment:
Shorter wait method for usage when waiting on a UI element in vscode e.g. wait for the command palette to open.